specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/elasticsan.json

```diff
@@ -1170,12 +1388,21 @@
         }
       }
     },
-    "ElasticSan": {
+    "ElasticSanCreateParameter": {
       "type": "object",
       "description": "Response for ElasticSan request.",
```

Comment:
The description looks not right. This should be input parameter, instead of responds from server.